backend/expense/README.md

```
@@ -1,26 +1,74 @@
-- Works with an expense calculator that contains the business logic to compute bills. 
-
-### API Specs
-https://raw.githubusercontent.com/egovernments/DIGIT-Specs/master/works/expense-contract.yml
\ No newline at end of file
+# Expense Service
+
+The expense module implements the functionality of bills and payments. A bill or a group of bills can be aggregated together as payments.
+The expense module always works in combination with a calculator service.
+The calculator calls into the expense service to create bills.
+
+### Service Dependencies
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)